src/nvapi_d3d.cpp

```diff
@@ -36,4 +56,38 @@ extern "C" {
 
         return Ok(n);
     }
+
+    NvAPI_Status __cdecl NvAPI_D3D1x_GetGraphicsCapabilities(IUnknown *pDevice,
+                                                    NvU32 structVersion,
+                                                    NV_D3D1x_GRAPHICS_CAPS *pGraphicsCaps) {
+        constexpr auto n = __FUNCTION__;
+        switch(structVersion) {
```

Comment:
Please format the `switch` like here: https://github.com/jp7677/dxvk-nvapi/blob/master/src/nvapi_gpu.cpp#L195